atom/browser/native_window.cc

```diff
@@ -94,7 +94,7 @@ NativeWindow::NativeWindow(
   // add to browser list
   ::Browser::CreateParams create_params(::Browser::Type::TYPE_TABBED,
       Profile::FromBrowserContext(inspectable_web_contents_->
-          GetWebContents()->GetBrowserContext()));
+          GetWebContents()->GetBrowserContext()), false);
```

Comment:
always setting this to false seems problematic because it ignores the actual user gesture status. Where is this used?